ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -427,7 +427,7 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )gcp-e2e-operator,?($|\s.*)
+    trigger: (?m)^/test( | .* )(gcp-e2e-operator|remaining-required),?($|\s.*)
```

Comment:
If for whatever reason bot fails to execute tests, you can manually issue /test remaining-required. Also bot does not handle retests if your pipline fails either a user needs to step in (as usual) or retester will try to handle if all labels are met.